test/lit/passes/optimize-instructions-gc-iit.wast

```diff
@@ -3,16 +3,22 @@
 ;; RUN:   | filecheck %s
 ;; RUN: wasm-opt %s --optimize-instructions --ignore-implicit-traps --enable-reference-types --enable-gc --nominal -S -o - \
 ;; RUN:   | filecheck %s --check-prefix NOMNL
+;; Also test trapsNeverHappen (with nominal; no need for both type system modes).
+;; RUN: wasm-opt %s --optimize-instructions --traps-never-happen --enable-reference-types --enable-gc --nominal -S -o - \
+;; RUN:   | filecheck %s --check-prefix TNHNL
```

Comment:
I would probably go with NOMNL-TNH or something even more verbose here to prioritize clarity over matching character counts.

Reply:
Sounds good, done.